### DISCUSSION TITLE
Suggestions for the profile page

### DISCUSSION THREAD

**Discussion by ghost** [deso-protocol/dips#64] [Medium CIPs]
Blog section
Required for very long articles of the creator. In this way, Bitclout members will not need platforms such as medium and wordpress. And creators will be able to earn income for these articles as well.

Memberships on other platforms Section
An automatic verification system should be created for Bitclout. The system will be like this. For example, I have an Instagram account. I am adding the code generated in bitclout.com to my bio on my instagram account. And this automated system automatically authenticates my identity. This is an absolutely essential feature.

**Comment by tijno** [comment]
i think this is already covered in another cip - ill find the link.

**Reply by tijno** [reply]
https://github.com/bitclout/cips/discussions/28

**Reply by ghost** [reply]
Yes, I just saw it. thank you. The about section has already been suggested. I'm deleting it
